server/utils/generateRoomCode.ts

```diff
@@ -18,7 +16,8 @@ const generateRoomCode = async (rooms: Collection<RoomSchema>): Promise<string>
 
 const getRandomCode = () => {
   let text = '';
-  const possible = 'AB';
+  // const possible = 'ABCDEFGHIJKLMNOPQRSTUVWXYZ';
```

Comment:
Should `possible` still be like this?

Reply:
I left it for testing (in order to be able to easily create all possible combinations and therefore simulate duplication events), but I have no problem updating it.

Reply:
Up to you. I'm fine either way just wanted to point it out in case it was left by accident.